scipy/stats/tests/test_mstats_basic.py

```diff
@@ -454,6 +454,19 @@ def test_winsorization(self):
         winsorized = mstats.winsorize(data)
         assert_equal(winsorized.mask, data.mask)
 
+    def test_winsorization_nan(self):
+        data = ma.array([np.nan, np.nan, 0, 1, 2])
+        assert_raises(ValueError, mstats.winsorize, data, (0.05, 0.05)
+                      , nan_policy='raise')
```

Comment:
Looks like the whitespace around the comma ` , ` may be causing the linter check in CI.

I sometimes get caught by the linter too. If you want to check locally before pushing:

```
pip install pycodestyle==2.5.0
pycodestyle scipy benchmarks/benchmarks
```

Reply:
Thank you, this is very helpful! I'll look into this tonight.

Reply:
I took care of the whitespace issue and I believe I have properly rebased on the latest scipy master branch. I truly appreciate the guidance here, I'm mostly familiar with the most basic/common git operations so I'm learning as I'm going.